apps/meeting/package.json

```diff
@@ -8,8 +8,8 @@
     "@types/webpack-env": "^1.15.1",
     "@typescript-eslint/eslint-plugin": "^2.19.2",
     "@typescript-eslint/parser": "^2.19.2",
-    "amazon-chime-sdk-component-library-react": "^2.9.0",
-    "amazon-chime-sdk-js": "^2.16.1",
+    "amazon-chime-sdk-component-library-react": "file:../../../../amazon-chime-sdk-component-library-react/amazon-chime-sdk-component-library-react-2.9.1.tgz",
```

Comment:
Did you make this change particularly?
Feels like we should change it to 2.9.1.

Reply:
A nice catch

Reply:
Oops, remnant from my previous testing with react library :D